wiki/People/Community_Contributors/de.md

```diff
@@ -227,6 +222,35 @@ Mitwirkende, die ihre Accounts während einer früheren Ära des Spiels registri
 | ::{ flag=US }:: [ChillierPear](https://osu.ppy.sh/users/9501251) | Herausragende Beiträge zur osu!-Turnierszene und den World Cups |
 | ::{ flag=NL }:: [cavoeboy](https://osu.ppy.sh/users/7361815) | Herausragende Beiträge zu osu! IRL-Events und zur Turnierszene |
 
+### 2023
+
+#### November
+
+*Für den Newsbeitrag, siehe: [Community Contributors: 2022 & 2023](https://osu.ppy.sh/home/news/2023-11-19-community-contributors-2022-2023)*
+
+| Benutzer | Beiträge |
+| :-- | :-- |
+| ::{ flag=AU }:: [Ephemeral](https://osu.ppy.sh/users/102335) | Außergewöhnliches und unermüdliches Engagement im Community-Management für über ein Jahrzent |
+| ::{ flag=PL }:: [Venix](https://osu.ppy.sh/users/5999631) | Herausragendes Engagement zur Moderationsszene und den [Beatmap Spotlights](/wiki/Beatmap_Spotlights) |
+| ::{ flag=CH }:: [TicClick](https://osu.ppy.sh/users/672931) | Herausragende Beiträge zu Moderationsangelegenheiten und dem osu!-Wiki (wikifriend) |
+| ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | Herausragende und wegweisende Beiträge zum osu!taiko-Spielmodus |
+| ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | Herausragende Führung zu Angelegenheiten im Beatmap-Management |
+| ::{ flag=US }:: [Cychloryn](https://osu.ppy.sh/users/6921736) | Herausragende Beiträge zur Mappingszene durch die Entwicklung von Tools ([osumod.com](https://osumod.com)) |
+| ::{ flag=US }:: [BTMC](https://osu.ppy.sh/users/3171691) | Herausragende Beiträge zu Offline-Turnieren und dem Wachstum der Community allgemein |
+| ::{ flag=CA }:: [D I O](https://osu.ppy.sh/users/3958619) | Herausragende Beiträge zur osu!-Turnierszene und den World Cups |
```

Comment:
Some time ago, respektive and I decided to use `World Cup` instead of `Weltmeisterschaft` (I assume due to a longer term containing "World Cup" which doesn't sound well when translated). Should this still be a thing? 🤔

In my opinion it's fine. I wanted to ask what others think of this.